app/src/at-tamle/java/asiantech/internship/summer/KotlinW2Json/Main.kt

```
@@ -0,0 +1,211 @@
+        val sortBy = menu()
+        val n = Com.input("số lượng cần hiển thị: ")
+        when (sortBy) {
+            1 -> sortStudentByName(students, n)
+            2 -> sortStudentByBirthDay(students, n)
+            3 -> sortStudentByStId(students, n)
+            4 -> sortStudentByDegreeNo(students, n)
+            5 -> sortStudentByID(students, n)
+        }
+    }
+
+    private fun menu(): Int {
+        println(
+                "1. Sort by name\n" +
+                        "2. Sort by birthday\n" +
+                        "3. Sort by student ID\n" +
+                        "4. Sort by degreeNo\n" +
+                        "5. Sort by list ID\n"
+        )
+        return Com.input("Nhập vào số tương ứng: ")
+    }
+
+    private fun sortStudentByID(students: List<Student>, n: Int) {
+        val studentsSorted = students.sortedBy { it.id }
+        if (n < studentsSorted.size) {
+            for (i in 0 until n) {
+                println(studentsSorted[i].id)
+            }
+        } else {
+            studentsSorted.forEach { println(it) }
+        }
+    }
+
+    private fun sortStudentByDegreeNo(students: List<Student>, n: Int) {
+        val studentsSorted = students.sortedBy { it.degree_no }
+        if (n < studentsSorted.size) {
+            for (i in 0 until n) {
+                println(studentsSorted[i].degree_no)
+            }
+        } else {
+            studentsSorted.forEach { println(it.degree_no) }
+        }
+    }
+
+    private fun sortStudentByStId(students: List<Student>, n: Int) {
+        val studentsSorted = students.sortedBy { it.student_id }
+        if (n < studentsSorted.size) {
+            for (i in 0 until n) {
+                println(studentsSorted[i].student_id)
+            }
+        } else {
+            studentsSorted.forEach { println(it.student_id) }
+        }
+    }
+
+    private fun sortStudentByBirthDay(students: List<Student>, n: Int) {
+        val sdf = SimpleDateFormat("dd/MM/yyyy")
```

WARNING: [SimpleDateFormat] To get local formatting use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()`, or use `new SimpleDateFormat(String template, Locale locale)` with for example `Locale.US` for ASCII dates.
 Almost all callers should use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()` to get a ready-made instance of SimpleDateFormat suitable for the user's locale. The main reason you'd create an instance this class directly is because you need to format/parse a specific machine-readable format, in which case you almost certainly want to explicitly ask for US to ensure that you get ASCII digits (rather than, say, Arabic digits).

Therefore, you should either use the form of the SimpleDateFormat constructor where you pass in an explicit locale, such as Locale.US, or use one of the get instance methods, or suppress this error if really know what you are doing.